### DISCUSSION TITLE
Conditionally display RSVP field based on custom event field

### DISCUSSION THREAD

**Discussion by TimB111** [liedekef/events-made-easy#173] [General]
When creating an event I want to have an option whether this event should require guests to make a choice from various options and type their choices into a textarea field. I have the following custom fields:
- "evnt_guest_choices_required", type=checkbox, required=no, purpose=events, used in replies=yes, field values="y"
- "rsvp_guest_choices", type=textarea, required=no, purpose=rsvp, used in replies=no

In EME > Settings > RSVP > Booking form format, I'm using this but it doesn't work even though the "evnt_guest_choices_required" checkbox is ticked on the edit event page:
`[eme_if tag='#_FIELD{evnt_guest_choices_required}' value='y']<div class="row"><div><label>Guest choices</label></div><div>#_FIELD{rsvp_guest_choices}</div></div>[/eme_if]`

Am I doing something wrong here?

**Comment by liedekef** [comment]
Using this (bug) changeset: https://github.com/liedekef/events-made-easy/commit/8ae4c942fc56898fb202d86214cdd96afca9a63f , there will be support for #_EVENTFIELD and then your code will work.
Reason that it doesn't work here is that in fact inside the RSVP form, the form for the field is shown (also for events) and not its value (a code inspection would get you there, or just do #_FIELD{evnt_guest_choices_required} without the surrounding eme_if )

**Reply by TimB111** [reply]
Great, thank you, I'll try this out.

**Reply by liedekef** [reply]
And this changeset might help too: https://github.com/liedekef/events-made-easy/commit/442d69df64d86a2a52d1be5126d8ee8a14e62ffe
==> smaller than the first changeset and can be applied independently.

**Reply by TimB111** [reply]
Thank you
